neqo-qpack/src/encoder.rs

```diff
@@ -590,7 +590,11 @@ mod tests {
     fn connect_generic(huffman: bool, max_data: Option<u64>) -> TestEncoder {
         let mut conn = default_client();
         let mut peer_conn = if let Some(max) = max_data {
-            default_server_max_stream_data(max)
+            default_server_max_stream_data(
```

Comment:
Can you change the name of this?  `configure_server()` or something like that might be better.  It's not entirely a default now.

Reply:
I should not write code after 11PM after a long day